KuringApp/KuringApp/Setting/AppIconSelectView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  AppIconSelectView.swift
+//  KuringApp
+//
+//  Created by 박성수 on 2023/09/27.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+enum KuringIcon: String, CaseIterable {
```

Comment:
```suggestion
enum KuringIcon: String, CaseIterable, Identifiable {
    var id: String { self.rawValue }
```

`List`에서 `id` 파라미터 안쓰기 위함